2022.md

```diff
@@ -1,5 +1,8 @@
 # 2022 Blogs
 
+## August
```

Comment:
Shall we add the blog count as well?
```suggestion
## August (1)
```

Reply:
Fxed wth : [8489a7f](https://github.com/wso2/iam-blogs/pull/162/commits/8489a7f0caac203c412f09061f505ec3458d9d71)